assets/js/base/hooks/cart/use-store-cart-item-quantity.js

```diff
@@ -62,9 +77,29 @@ export const useStoreCartItemQuantity = ( cartItem ) => {
 			changeCartItemQuantity( cartItemKey, debouncedQuantity );
 		}
 	}, [ debouncedQuantity, cartItemKey ] );
+	useEffect( () => {
+		if ( isPendingQuantity ) {
```

Comment:
It looks like there are no components that use `isPendingQuantity` - do we need it in the store, or is it just used to for the `incrementCalculating`/`decrementCalculating`. Curious if we can simplify that code, since the row component only needs `isPendingDelete`, and cart overall needs `isCalculating`.

Reply:
we need it for the `incrementCalculating`/`decrementCalculating` yeah, currently I see no way to communicate between action dispatchers and our hook except via the store, I would be happy to reduce this it's possible.